v3/src/components/case-card/case-view.tsx

```diff
@@ -1,46 +1,112 @@
 import { IAttribute } from "../../models/data/attribute"
+import { ICollectionModel } from "../../models/data/collection"
+import { IDataSet } from "../../models/data/data-set"
+import { IGroupedCase } from "../../models/data/data-set-types"
 import { createAttributesNotification } from "../../models/data/data-set-notifications"
 import { uiState } from "../../models/ui-state"
 import { uniqueName } from "../../utilities/js-utils"
-import { IDataSet } from "../../models/data/data-set"
-import { colorCycleClass } from "../case-tile-common/case-tile-utils"
+import { t } from "../../utilities/translation/translate"
+import { urlParams } from "../../utilities/url-params"
+import { CaseAttrsView } from "./case-attrs-view"
 import { CaseCardCollectionSpacer } from "./case-card-collection-spacer"
 import { CaseCardHeader } from "./case-card-header"
+import { useCaseCardModel } from "./use-case-card-model"
 
 import AddIcon from "../../assets/icons/add-data-icon.svg"
 
 import "./case-view.scss"
 
+const animationDuration = 300
+
 interface ICaseViewProps {
   cases: IGroupedCase[]
+  dummy?: boolean // Dummy case views are used for animations and have stripped down functionality
+  onNewCollectionDrop?: (dataSet: IDataSet, attrId: string, beforeCollectionId: string) => void
   level: number
-  onSelectCases: (caseIds: string[]) => void
-  displayedCaseLineage?: readonly string[]
-  onNewCollectionDrop: (dataSet: IDataSet, attrId: string, beforeCollectionId: string) => void
+  onSelectCases?: (caseIds: string[]) => void
+}
+
+interface IRenderSingleCaseViewArgs {
+  displayedCase?: IGroupedCase
+  dummy?: boolean
+  style?: React.CSSProperties
 }
 
-export const CaseView = observer(function InnerCaseView(props: ICaseViewProps) {
-  const {cases, level, onSelectCases, onNewCollectionDrop, displayedCaseLineage = []} = props
+export const CaseView = observer(function InnerCaseView({
+  cases, dummy, level, onNewCollectionDrop, onSelectCases
+}: ICaseViewProps) {
   const cardModel = useCaseCardModel()
+  const tileLayout = useFreeTileLayoutContext()
   const data = cardModel?.data
   const collectionCount = data?.collections.length ?? 1
   const collectionId = useCollectionContext()
   const collection = data?.getCollection(collectionId)
-  const initialSelectedCase = collection?.cases.find(c => c.__id__ === displayedCaseLineage[level])
-  const displayedCase = initialSelectedCase ?? cases[0]
-  const displayedCaseId = displayedCase?.__id__
+  const selectedCaseIndices = data?.selectedCaseIndices ?? []
+  const collectionSelectedCaseIndices = selectedCaseIndices[level] ?? []
+  const displayedCaseIndex = collectionSelectedCaseIndices[0] ?? -1
+  const previousSelectedCaseIndex = useRef<number>(displayedCaseIndex)
+  const displayedCaseId = collection?.caseIds[displayedCaseIndex] ?? ""
+  const displayedCase = data?.caseInfoMap.get(displayedCaseId)?.groupedCase ?? cases[0]
+
+  const disableAnimation = urlParams.noComponentAnimation != null
```

Comment:
As I mentioned in slack, I don't think it's necessary for `noComponentAnimation` to control this internal animation, but I'm also not opposed to it, so it's fine for now.

Reply:
Ah yeah, I was going to comment about this on slack but I never got around to it. I had already implemented this before you posted--it's only three lines or so. And it turns out it is helpful in getting one of the cypress tests to pass that was proving to be a pain to fix without it.